docs/.storybook/configs/theme-context.js

```diff
@@ -1,20 +1,10 @@
 import PropTypes from 'prop-types';
 import { designTokens, ThemeProvider } from '../../../design-system';
 
-const defaultTheme = designTokens;
```

Comment:
Now that we're working with the new theme, I updated Storybook to actually use it instead of a custom one defined here.